java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -6801,4 +6805,58 @@ public void restore_with_restoreOptions_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(OK, response.get());
     }
+
+    @SneakyThrows
+    @Test
+    public void sscan_returns_success() {
+        // setup
+        String key = "testKey";
+        long cursor = 0;
+        String[] arguments = new String[] {key, Long.toString(cursor)};
+        Object[] value = new Object[] {0L, new String[] {"hello", "world"}};
+
+        CompletableFuture<Object[]> testResponse = new CompletableFuture<>();
+        testResponse.complete(value);
+
+        // match on protobuf request
+        when(commandManager.<Object[]>submitNewCommand(eq(SScan), eq(arguments), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Object[]> response = service.sscan(key, cursor);
+        Object[] payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(value, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void sscan_with_options_returns_success() {
```

Comment:
why can't it just be named sscan_with_options ?

Reply:
Just following established naming convention in the file.